go.mod

```diff
@@ -3,27 +3,29 @@ module github.com/firecracker-microvm/firecracker-containerd
 require (
 	github.com/Microsoft/go-winio v0.5.0 // indirect
 	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705 // indirect
-	github.com/awslabs/tc-redirect-tap v0.0.0-20200708224642-a0300978797d
+	github.com/awslabs/tc-redirect-tap v0.0.0-20211025175357-e30dfca224c2
 	github.com/bits-and-blooms/bitset v1.2.1 // indirect
 	github.com/containerd/containerd v1.5.9
 	github.com/containerd/continuity v0.2.0 // indirect
 	github.com/containerd/fifo v1.0.0
+	github.com/containerd/go-cni v1.1.1 // indirect
 	github.com/containerd/go-runc v1.0.0
 	github.com/containerd/ttrpc v1.1.0
 	github.com/containerd/typeurl v1.0.2
-	github.com/containernetworking/cni v0.8.1
-	github.com/containernetworking/plugins v0.9.1
-	github.com/firecracker-microvm/firecracker-go-sdk v0.22.1-0.20210520223842-abd0815b8bf9
+	github.com/containernetworking/cni v1.0.1
```

Comment:
@kzys closed a PR to update this in #561, Does that mean this is also blocked on updating containerd?

Reply:
It may be. I'm not 100% sure though.

Reply:
Regarding upgrading `containerd` to `1.6.0` release which is only compatible with `golang 1.16` upwards, do we want to do the same? What's the consensus here?

Reply:
BTW I've created #569. This will be addressed by a standalone PR.